oop/taxiPark.js

```diff
@@ -0,0 +1,89 @@
+class Car {
```

Comment:
Please add `;` everywhere where it's necessary

Reply:
add ; everywhere